build/helper/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM debian:buster-slim
```

Comment:
Wouldn't it be lighter and more straightforward to build it from `alpine` ?
We could maybe build this image in an CI/CD pipeline to users only need to pull it.

Reply:
I've tested with success this
```
FROM alpine:3.10

RUN set -ex; \
    # Install Bash and OpenSSL
    apk add --no-cache bash openssl
```
The image is ~10MB compared to ~75MB with Debian. We should also maybe consider not running it as root and provide a USER_ID when run the script.

Reply:
changed to alpine